src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -323,6 +324,29 @@ public String getNoCausesMessage() {
         return noCausesMessage;
     }
 
+    /**
+     * Whether to display in the build page when no causes are identified.
+     *
+     * @return true if on.
+     */
+    public boolean isNoCausesEnabled() {
+        if (noCausesEnabled == null) {
+            return true;
+        } else {
+            return noCausesEnabled;
+        }
+    }
+
+    /**
+     * Sets whether the "no indications found" message should be shown in the job page when no causes are found.
+     * Default value is true.
+     *
+     * @param noCausesEnabled on or off.
+     */
+    public void setNoCausesEnabled(boolean noCausesEnabled) {
```

Comment:
```suggestion
    @DataBoundSetter
    public void setNoCausesEnabled(boolean noCausesEnabled) {
```

Reply:
It looks like things work fine without the annotation, see test in 80c4cfd. 

Of course, I agree the setter could be annotated (better being explicit), but then there are several other setters which also lack the same annotation (`setDoNotAnalyzeAbortedJob`, `setGlobalEnabled`, `setTestResultParsingEnabled`, `setTestResultCategories`, `setGerritTriggerEnabled`, `setNrOfScanThreads`, `setMaxLogSize`).

Do you mind if I "fix" them all in one more commit to this PR?  Or do you prefer me fixing `setNoCausesEnabled` only here, and then a new PR for the other ones?